provision/acc_provision/acc_provision.py

```diff
@@ -202,19 +202,19 @@ def config_discover(config, prov_apic):
     if prov_apic is not None:
         apic = get_apic(config)
 
+    orig_infra_vlan = config["net_config"].get("infra_vlan")
```

Comment:
LGTM. I hope value setting to None wont create problem in the code below where it is referenced.

Reply:
There is a use case where None is a valid value, so we need to handle that. And the following code should handle it.